driver-core/src/test/java/com/datastax/driver/core/policies/LoggingRetryPolicyIntegrationTest.java

```diff
@@ -54,7 +55,7 @@ public void setUpRetryPolicy() {
         setRetryPolicy(new LoggingRetryPolicy(new CustomRetryPolicy()));
     }
 
-    @BeforeMethod(groups = {"short", "unit"})
```

Comment:
Cluster is not initialized in group "unit", that's why cluster.getConfiguration() below results in NPE.